src/timedata.cpp

```diff
@@ -82,12 +82,9 @@ void AddTimeData(const CNetAddr& ip, int64_t nOffsetSample, int nOffsetLimit)
             LogPrintf("*** %s\n", strMessage);
             uiInterface.ThreadSafeMessageBox(strMessage, "", CClientUIInterface::MSG_ERROR);
         }
-        if (fDebug) {
-            for (int64_t n : vSorted)
-                LogPrintf("%+d  ", n);
-            LogPrintf("|  ");
-        }
-        LogPrintf("nTimeOffset = %+d\n", nTimeOffset);
+        for (int64_t n : vSorted)
```

Comment:
Shouldn't be under a `if (logCategories != BCLog::NONE)` block?

Reply:
Yes.
Just thought that it would be good to have this logged by default, given the importance of the time offset.
If not, and we want to keep the same functionality as before, we can adjust in #1459 (where `logCategories` gets encapsulated in the logger class) and replace it with `!GetBoolArg("-shrinkdebugfile", g_logger->DefaultShrinkDebugFile())`.